Cargo.toml

```diff
@@ -47,8 +48,182 @@ exclude = [
     "mm2src/mm2_test_helpers",
 ]
 
-# https://doc.rust-lang.org/beta/cargo/reference/features.html#feature-resolver-version-2
```

Comment:
I see you removed this doc comment but we can instead write a comment on why we use version 2

Reply:
This is pretty the standard. Reverted the change